README.md

```diff
@@ -103,7 +103,7 @@ To install the MapboxNavigation framework using [Carthage](https://github.com/Ca
    # Latest stable release
    github "mapbox/mapbox-navigation-ios" ~> 2.3
```

Comment:
We've released 2.4.0 from branch `release-2.4.0`. We should update this readme for the stable version. 
Also `CFBundleVersion == 100` is used in both `2.4.0` and `2.5.0-beta.1`. Not sure if it's an issue for us.

Reply:
Good catch. Will update latest stable release versions. `CFBundleVersion` should not be a problem.